compiler+runtime/src/cpp/jank/codegen/llvm_processor.cpp

```diff
@@ -1083,6 +1112,64 @@ namespace jank::codegen
     return ctx->builder->CreateLoad(ctx->builder->getPtrTy(), global);
   }
 
+  llvm::Value *
+  llvm_processor::gen_var_root(obj::symbol_ref qualified_name, var_root_kind kind) const
+  {
+    auto it(ctx->var_root_globals.find(qualified_name));
+    if(it != ctx->var_root_globals.end())
+    {
+      /* If the var-root already exists, and is part of a normal var-deref, create a load in the IR
+       * Else if part of the special cases, just return the value of the var-root directly
```

Comment:
This comment has multiple issues.

1. It's describing *what* the code is doing, not *why*
2. The second line, in particular, is not a coherent sentence, since it's written as pseudo code
3. None of the sentences in the comments here and the rest of this function end in periods

Please treat comments as documentation, like you would read in a book or user guide. Sentences should be clear, use correct grammar and spelling, have correct punctuation, and have a clear meaning for being in the guide in the first place.

This is likely to be dissimilar to comments you have seen in your experience coding so far. Those comments were generally pretty awful. Please revisit ALL of your comments in this PR and write them with this in mind. **Explain clearly.**